presto/dbapi.py

```diff
@@ -236,10 +245,111 @@ def setinputsizes(self, sizes):
     def setoutputsize(self, size, column):
         raise presto.exceptions.NotSupportedError
 
-    # TODO support params
-    def execute(self, operation):
-        self._query = presto.client.PrestoQuery(self._request, sql=operation)
-        result = self._query.execute()
+    def _execute_prepare_statement(self, operation, statement_name):
```

Comment:
Maybe rename to `prepare_statement` since we don't actually `EXECUTE` here.

Reply:
Sounds good. I'll do the same to with the deallocate part of it.

Reply:
Done